### ISSUE TITLE
Default_operator ‘AND’ with query_string and multiple fields does not work properly

### ISSUE THREAD

**Issue by Flinou** [elastic/elasticsearch#48860]
**Elasticsearch version** (`bin/elasticsearch --version`): 

> 7.2.0

**JVM version** (`java -version`):

> 
> openjdk version "11.0.1" 2018-10-16
> OpenJDK Runtime Environment 18.9 (build 11.0.1+13)
> OpenJDK 64-Bit Server VM 18.9 (build 11.0.1+13, mixed mode)

**OS version** (`uname -a` if on a Unix-like system):

> Linux 4b495c31e063 3.10.0-862.9.1.el7.x86_64 #1 SMP Mon Jul 16 16:29:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Description of the problem including expected versus actual behavior**:

I have some issues trying to make default_operator 'AND' work with query_string on multiple fields. 
I have an index containing a document with those fields  : 

> "content" : "attention pour la revision"
> "main_title" : "voiture est dans le garage"

If I launch the request below : 
```
GET testeur/_search
    {
      "query": {
        "query_string": {
          "query": "attention voiture",
          "default_operator": "AND",
          "fields": ["content","main_title"]
        }
      }
   }

```
The document is not retrieved. This behaviour does not seem consistent.
The document is retrieved specifying the operator AND directly in the query field though. 

Example : 
```
GET testeur/_search
{
  "query": {
    "query_string": {
      "query": "attention AND voiture",
      "fields": ["content","main_title"]
    }
  }
}
```
**Steps to reproduce**:

 1. Document creation : 
```
POST testeur/_doc/document
{
  "content": "attention dans la revision",
  "main_title" : "fichier dans la voiture"
}
```

 2. Get with default_operator AND
```
GET testeur/_search
    {
      "query": {
        "query_string": {
          "query": "attention voiture",
          "default_operator": "AND",
          "fields": ["content","main_title"]
        }
      }
    }
```

 3. Get with AND directly in the "query" field : 
```
GET testeur/_search
{
  "query": {
    "query_string": {
      "query": "attention AND voiture",
      "fields": ["content","main_title"]
    }
  }
}
```

**Comment by javanna** [comment]
We reserve github for confirmed but reports and feature requests. I think your question is better suited for our forums at https://discuss.elastic.co/. 

Some general feedback, but please let's take this to the forums if it needs more discussion. Have you tried using the explain API to try and see how the query changes internally in the two described scenarios? There are subtleties when it comes to searching for multiple terms against multiple fields. Where is the operator applied? Do all terms need to appear in all fields? For things like this, match query allows you to configure its behaviour and is generally better than query_string.

**Comment by Flinou** [comment]
I did post on discuss.elastic : [Link](https://discuss.elastic.co/t/default-operator-and-with-query-string-does-not-work-properly/203667). Unfortunately, I got no answers. 

I did set profile to true using the query_string. 

> With default_operator : **"description" : "((+content:attention+main_title:voiture) | (+content:attention+main_title:voiture)**


> With And directly set in query : **"description" : ""+(content:attention| main_title:attention) +(content:voiture| main_title:voiture)"**

Reading the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html), there is nothing explaining this difference. Thus, I think this behaviour is not consistent and this is actually a bug.
